netbench/cdk/src/main/java/com/aws/EcsStack.java

```diff
@@ -108,7 +115,9 @@ public EcsStack(final Construct parent, final String id, final EcsStackProps pro
 
             LogGroup serviceLogGroup = LogGroup.Builder.create(this, "server-log-group")
                 .retention(RetentionDays.ONE_DAY)
-                .logGroupName("server-logs" + new SimpleDateFormat("dd-MM-yyyy-HH-mm-ss").format(new Date()).toString())
+                .logGroupName("server-logs" + new SimpleDateFormat("MM-dd-yyyy-HH-mm-ss").format(new Date()).toString())
+                .removalPolicy(RemovalPolicy.DESTROY)
```

Comment:
what does this do?

Reply:
It helps clean up the log group when the CDK application is destroyed so that the log groups don't start building up. All logs are saved into an s3 bucket anyway, so I didn't want to keep the cloudwatch logs around unnecessarily.